### ISSUE TITLE
PodCIDR is not present on Azure CNI configurations

### ISSUE THREAD

**Issue by sozercan** [Azure/aad-pod-identity#40]
PodCIDR is not present on default configuration of acs-engine (using Azure CNI) and advanced networking configuration of AKS (uses Azure CNI instead of kubenet)

Deploying will result in: (for acs-engine)

```
time="2018-06-06T18:52:10Z" level=info msg="starting nmi process"
time="2018-06-06T18:52:10Z" level=info msg="Listening on port 2579"
time="2018-06-06T18:52:10Z" level=info msg="node: k8s-agentpool-35881120-vmss000000"
time="2018-06-06T18:52:10Z" level=fatal msg="podcidr is nil or empty, nodename: k8s-agentpool-35881120-vmss000000"
```

**Comment by rite2nikhil** [comment]
Thanks for reporting the issue.

**Comment by rite2nikhil** [comment]
closed #62
